esm/html/script-element.js

```diff
@@ -1,18 +1,96 @@
-import {registerHTMLClass} from '../shared/register-html-class.js';
+import { booleanAttribute, stringAttribute } from '../shared/attributes.js'
+import { registerHTMLClass } from '../shared/register-html-class.js'
 
-import {TextElement} from './text-element.js';
+import { TextElement } from './text-element.js'
 
-const tagName = 'script';
+const tagName = 'script'
 
 /**
  * @implements globalThis.HTMLScriptElement
  */
 class HTMLScriptElement extends TextElement {
   constructor(ownerDocument, localName = tagName) {
-    super(ownerDocument, localName);
+    super(ownerDocument, localName)
+  }
+
+  get innerHTML() {
+    return super.innerHTML || ''
+  }
+  set innerHTML(value) {
+    super.textContent = value
+  }
+
+  get innerText() {
+    return super.innerText || ''
+  }
+  set innerText(value) {
+    super.textContent = value
+  }
+  get textContent() {
+    return super.textContent || ''
+  }
+  set textContent(value) {
```

Comment:
these look all redundant ... if you have to use getters and setters from super just *don't* implement that?